extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/OpenConnections.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.websockets.next;
+
+import java.util.Collection;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import io.smallrye.common.annotation.Experimental;
+
+/**
+ * Provides convenient access to all open connections from clients to {@link WebSocket} endpoints on the server.
+ * <p>
+ * Quarkus provides a built-in CDI bean with the {@link jakarta.inject.Singleton} scope that implements this interface.
+ */
+@Experimental("This API is experimental and may change in the future")
+public interface OpenConnections {
+
+    /**
+     * Returns an immutable snapshot of all open connections at the given time.
+     *
+     * @return an immutable collection of all open connections
+     */
+    default Collection<WebSocketConnection> get() {
+        return stream().collect(Collectors.toUnmodifiableList());
+    }
+
+    /**
+     * Returns the open connection with the given id.
+     *
+     * @param connectionId
+     * @return the open connection or {@code null} if no open connection with the given id exists
+     * @see WebSocketConnection#id()
+     */
+    default WebSocketConnection get(String connectionId) {
```

Comment:
```suggestion
    default Optional<WebSocketConnection> get(String connectionId) {
```

Reply:
Maybe renaming to `findByConnectionId` could be a good idea too, to indicate the purpose